_includes/footer.html

```diff
@@ -80,7 +72,7 @@
         <div class="bottom_footer">
             <div class="footer-copyright col-xs-12 col-md-8">
                 <p class="copyright">
-                    Copyright &copy; 2019 Docker Inc. All rights reserved. </p>
+                    Copyright &copy; 2020 Docker Inc. All rights reserved. </p>
```

Comment:
If we _do_ publish years, this should probably be 2013-2020

Not sure how the `All rights reserved` portion relates with the license (which is Apache 2 for the documentation source)

Reply:
Haven't changed the text here. The format is also inline with the copyrights format on docker.com. 

@lisaberry Could you provide some guidance here? Thanks.

Reply:
2013-2020.  Leave the other text in

Reply:
Updated the years. Thank you both!